packages/gatsby-plugin-google-gtag/src/gatsby-ssr.js

```diff
@@ -8,23 +8,26 @@ exports.onRenderBody = (
   if (process.env.NODE_ENV !== `production` && process.env.NODE_ENV !== `test`)
     return null
 
+  const gtagConfig = pluginOptions.gtagConfig || {}
+  const pluginConfig = pluginOptions.pluginConfig || {}
+
+  const selfHostedOrigin =
```

Comment:
Same minor comment as above - could we make this variable name just "origin" ? Defaulting a "self hosted" variable to google's domain could be confusing.